src/runtime/pkg/types/types_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2020 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+package types
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestKataRuntimeNameRegexp(t *testing.T) {
+	assert := assert.New(t)
+
+	runtimeNameRegexp, err := regexp.Compile(KataRuntimeNameRegexp)
+	assert.NoError(err)
+
+	assert.Equal(true, runtimeNameRegexp.MatchString("io.containerd.kata.v2"))
+	assert.Equal(true, runtimeNameRegexp.MatchString("io.containerd.kataclh.v2"))
+	assert.Equal(true, runtimeNameRegexp.MatchString("io.containerd.kata-clh.v2"))
```

Comment:
Might be worth adding in a test that includes periods in the Kata block. Something like:

```go
assert.Equal(true, runtimeNameRegexp.MatchString("io.containerd.kata.1.2.3-clh.4.v2"))
```

Reply:
Hi @jodh-intel , I have added it.